cypress/integration/components/generator.spec.js

```diff
@@ -1,31 +1,33 @@
+import { isOn, skipOn } from '@cypress/skip-test';
+
 describe('Generator component', () => {
     beforeEach(() => {
         cy.visit('/generator');
     });
 
     it('has generated a blob URL', () => {
-        cy.get('.request-transport-medium-chooser')
-            .contains('Fax')
-            .click();
+        cy.get('.request-transport-medium-chooser').contains('Fax').click();
 
-        cy.contains('Download PDF')
+        // Now that the PDF worker is only loaded on demand, the default timeout of 4 seconds is cutting it fairly
+        // close.
+        cy.contains('Download PDF', { timeout: 10000 })
             .should('not.have.class', 'disabled')
             .should('have.attr', 'href')
             .and('match', /^blob:https?:\/\/[\S]+?\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$/);
     });
 
     it('did not load pdfworker for email', () => {
-        cy.window().then(win => {
+        cy.window().then((win) => {
```

Comment:
Do we have access to `cy.window()` in production?

Reply:
Yes. Why wouldn't we?

Reply:
I guess this should be skipped in production, according to https://github.com/datenanfragen/website/pull/368/files#r484395914

Reply:
Since it's never set, it will always be `undefined` and the test will always pass. But sure, whatever.